src/patternfly/components/Switch/switch-label.hbs

```diff
@@ -0,0 +1,5 @@
+<span class="pf-c-switch__label{{#if switch-label--modifier}} {{switch-label--modifier}}{{/if}}"
+  {{#if switch-label--attribute}}
+    {{{switch-label--attribute}}}
+  {{/if}}>{{>@partial-block}}</span>
```

Comment:
nitpicky, but can you move `{{>@partial-block}}` and `</span>` to their own lines for all of these partials so they all have the same format as [switch.hbs](https://github.com/patternfly/patternfly-next/pull/724/files#diff-fe7c7ab175588e0b7d80ab8129a8ca27)?

Reply:
I did this on purpose.
This is harder to read:

```html
<label class="pf-c-switch" for="switch-on-example">
  <input class="pf-c-switch__input" type="checkbox" id="switch-on-example" aria-label="..." checked />
  <span class="pf-c-switch__label pf-m-off">
    Off</span>
  <span class="pf-c-switch__toggle"></span>
  <span class="pf-c-switch__label pf-m-on">
    On</span>
</label>
```

than

```html
<label class="pf-c-switch" for="switch-on-example">
  <input class="pf-c-switch__input" type="checkbox" id="switch-on-example" aria-label="..." checked />
  <span class="pf-c-switch__label pf-m-off">Off</span>
  <span class="pf-c-switch__toggle"></span>
  <span class="pf-c-switch__label pf-m-on">On</span>
</label>
```

And a span is an inline element. IMO better to keep the html element inlined

Reply:
Gotcha, it's no big deal, but it's the way most/all of the other partial files are formatted. Another way to do this (I think?) would be to use the formatting I suggested, and use [whitespace-control](https://handlebarsjs.com/expressions.html#whitespace-control). But I'll approve either way since we aren't currently enforcing a standard.